AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js

```diff
@@ -186,6 +199,15 @@ define([
             return record.canViewMoreKeywords();
         },
 
+        /**
+         * Drop all filters and initiate search on keyword click event
+         */
+        searchByKeyWord: function(keyword) {
```

Comment:
We need to drop any other filters that are currently active, I think it's better to forward the user to the URL instead of emulating user search.

Reply:
Hello @sivaschenko ,
Can I add the check if the filter is active or not? Personally, I don't think its possible to forward the URL with search key parameter because admin grid uses the `mui/index/render` url for updating the grid. let me know if you have any other idea.

Thanks

Reply:
Then it's needed to drop all the filters before the search

Reply:
Can you please check the current functionality in this video?
Please let me know if you want to make any changes
https://www.loom.com/share/b75ad2a739304e2ca0d8300d55aabaeb